### ISSUE TITLE
Mention in README that this is not working in Safari

### ISSUE THREAD

**Issue by vchrm** [kisenka/svg-sprite-loader#178]
Hi, the solution does not work in Safari :-(. The extraction works great, linking to sprite.svg#fragment from `background-image` CSS attribute is good with Chrome, but in Safari you have no luck.

Mentioning this in README would save me few hours implementing the package into our dev stack. We cannot afford having no icons in Safari.

Would it be a problem to mention this right away? Thank you.

**Comment by kisenka** [comment]
Hi! Could you please make a PR?

**Comment by kisenka** [comment]
@vchrm ping

**Comment by vchrm** [comment]
Yes, but not this week (too much work). Moreover I am still investigating the issue (trying to find some working polyfill or something).

**Comment by kisenka** [comment]
Check this one https://github.com/Keyamoon/svgxuse

**Comment by coderitual** [comment]
I have the same issue. Unfortunately svgxuse is not working for this purpose. Any other solution? Maybe this has something to do with the way svg-sprite-loader is creating fragments id's in external file (exposed by `use` element referenced internally to `symbol`).

**Comment by kisenka** [comment]
@vchrm currently it's a planned feature for 4.x (see kisenka/svg-baker#4), but as usual I have no time to implement it :) BTW the only working approach among all browsers is to create classic sprite and move it's `background-position`. I'll try to find time on next week.

**Comment by vchrm** [comment]
Do we agree that this is what Safari is missing? https://caniuse.com/#feat=svg-fragment

**Comment by kisenka** [comment]
@vchrm yes, both, desktop and mobile.

**Comment by coderitual** [comment]
The simplest solution is to use object instead of img. Not sure if this has some disadvantages.

**Comment by josemartos** [comment]
Looking forward to hearing from this 👍
